cli/command/trust/config.go

```diff
@@ -0,0 +1,17 @@
+package trust
+
+import (
+	"github.com/docker/cli/cli"
+	"github.com/docker/cli/cli/command"
+	"github.com/spf13/cobra"
+)
+
+func NewConfigCommand(dockerCli command.Cli) *cobra.Command {
```

Comment:
this is my fault because my fork doesn't have CI, but could we add golint style comments to these functions?  ex: `// NewConfigCommnand ...`

I'll work on adding CI